views/default/elgg.css.php

```diff
@@ -51,6 +52,7 @@
 echo elgg_view('elements/layout.css', $vars);
 echo elgg_view('elements/misc.css', $vars);
 echo elgg_view('elements/misc/spinner.css', $vars);
+echo elgg_view('elements/fa.css', $vars);
```

Comment:
I don't think we should move them at this point so elements/fa.css seems OK to me. What else do we need for this PR?